sorc/ncep_post.fd/CALHEL.f

```diff
@@ -285,10 +288,13 @@ SUBROUTINE CALHEL(DEPTH,UST,VST,HELI,USHR1,VSHR1,USHR6,VSHR6)
 !
 ! CASE WHERE THERE IS NO LEVEL WITH HEIGHT BETWEEN 5500 AND 6000
 !
+! Bo Cui 10/30/2019, remove "go to" statement
+
       DO J=JSTART,JSTOP
         DO I=ISTART,ISTOP
+          if(jcontinue) then
```

Comment:
I feel we don't need to have jcontinue

Reply:
I felt once it's out of 
ENDDO lloop
it should then continue and exit "if block" as there is no statements between 
ENDDO lloop
and 
END IF